charts/velero/templates/crds/cleanup-crds.yaml

```diff
@@ -1,39 +0,0 @@
-# This job is meant primarily for cleaning up on CI systems.
```

Comment:
I think it would be good to keep this file since it supports delete CRDs.

Reply:
Yes, I agree. We don't need to keep it inside the `crds` folder I don't think.

For reference:
https://github.com/vmware-tanzu/helm-charts/blob/efbcab45316125c5760a63cc75f2343879cf3fc0/charts/velero/ci/test-values.yaml#L26

Reply:
Good point, I revert it and placed in under `templates`